### PR TITLE
Prepare first release

### PR DESCRIPTION
**Pull request by robertodr**
This is infrastructure to manage releases in the future. For consistency, I suggest we follow the same branching model and tagging strategy as for MRCPP and MRChem.

- Added a workflow for automatic tagging with [Palisade](https://github.com/lightspeed/palisade/blob/master/doc/usage.md) Updated `CHANGELOG.md` _and_ `VERSION` will automatically tag and generate a release on GitHub with description obtained from the contents of the changelog.
- Added `.zenodo.json` @stigrj can you activate the magic that creates the DOI? I'll update the `README.md` accordingly.
- `__version__` is properly set according to contents of `VERSION`
- The version of MRCPP can be retrieved from Python by calling `mrcpp_version()`

### CODE REVIEW

**Comment by robertodr** [comment]
CI is probably FUBAR-ed until #38 is merged :facepalm:

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+[0.1.0]: https://github.com/MRChemSoft/vampyr/releases/tag/v0.1.0
+
+[Keep a Changelog]: https://keepachangelog.com/en/1.0.0/
+[Semantic Versioning]: https://semver.org/spec/v2.0.0.html
+
+[VAMPyR]: https://github.com/MRChemSoft/vampyr
+[MRCPP]: https://mrcpp.readthedocs.io
+[pybind11]: https://pybind11.readthedocs.io
+[Hylleraas Centre for Quantum Molecular Sciences]: http://www.ctcc.no/
```
Comment (stigrj) [inline CHANGELOG.md:29]: Wrong address

**Comment by stigrj** [comment]
Zenodo is activated

**Review by stigrj — APPROVED** [review]

**Comment by robertodr** [comment]
> Zenodo is activated

Do you have the DOI already?

**Comment by stigrj** [comment]
Looks like we need a release first

**Pull request merged**